cmd/sriovdp/main.go

```diff
@@ -57,6 +61,25 @@ func main() {
 		glog.Fatalf("Exiting.. one or more invalid configuration(s) given")
 		return
 	}
+
+	// Read global config
+	if err := config.Config.ReadConfig(cp.configFile); err != nil {
```

Comment:
Could you add an example of what the configMap looks like when `featureGates` data entry is added?
It looks like in the ReadConfig function, it is re-using the same cli cmd `configFile` which points to the resourceList data entry in configMap, but the umarshalling inside ReadConfig is failing when there is a resourceList data entry.

Reply:
Added the example as a part of manager_test, just for now: https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/26fce9ae66492aea18f6b355b0845d2556553fa1/cmd/sriovdp/manager_test.go#L118